src/components/Tool/Appeal/Modals/AddContactToAppealModal/AddContactToAppealModal.test.tsx

```diff
@@ -165,4 +173,46 @@ describe('AddContactToAppealModal', () => {
       });
     });
   });
+
+  it('adds an excluded contact and shows excluded message', async () => {
+    const { getByRole, getAllByText, findByRole, findByTestId, queryByTestId } =
+      render(<Components />);
+
+    userEvent.click(getByRole('combobox', { name: 'Contacts' }));
+    expect(await findByRole('option', { name: 'Alice' })).toBeInTheDocument();
+    userEvent.click(getByRole('option', { name: 'Alice' }));
```

Comment:
You can just click on the option instead of asserting its existence and clicking on it because the test will fail if it couldn't find the option, even without `expect().toBeInTheDocument()`.